src/app/component/MenuOverlay/MenuOverlay.style.scss

```diff
@@ -20,18 +21,35 @@
 
     &-Menu {
         padding: 1rem;
+        
+        li:last-child {
+            margin-bottom: 0;
+        }
     }
 
     &-Banner {
-        padding: 2rem 1rem;
-        text-align: center;
-        background-color: rgb(243, 137, 79);
-        font-size: 1.8rem;
-        color: white;
-        line-height: 1.3;
+        height: 120px;
+        display: flex;
+        justify-content: center;
+        margin-bottom: 12px;
 
-        &:last-of-type {
-            margin-bottom: .5rem;
+        picture {
+            padding-bottom: 0;
+            height: 100%;
+        }
+
+        img {
+            object-fit: cover;
+        }
+
+        figcaption {
```

Comment:
Could you please escape styling by element?

Reply:
It is not possible here, as it is a CMS block